server/http/helloapp/command.go

```diff
@@ -0,0 +1,153 @@
+package helloapp
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/mattermost/mattermost-plugin-apps/server/apps"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/httputils"
+	"github.com/mattermost/mattermost-plugin-apps/server/utils/md"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/pkg/errors"
+)
+
+type SubCommand struct {
+	AppID       string        `json:"app_id"`
+	Pretext     string        `json:"pretext"`
+	Description string        `json:"description"`
+	SubCommands []SubCommand  `json:"sub_commands"`
+	Args        []interface{} `json:"args"`
+	FormURL     string        `json:"form_url"`
+}
+
+func (h *helloapp) handleHelloCommandSubmission(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, call *apps.Call) (int, error) {
+	var e = func(err error, msg string) (int, error) {
+		httputils.WriteJSON(w, apps.CallResponse{
+			Type:     apps.CallResponseTypeError,
+			Markdown: md.MD(errors.Wrap(err, msg).Error()),
+		})
+		return http.StatusInternalServerError, err
+	}
+
+	username := call.Values.Get("user")
+	message := call.Values.Get("message")
+	u, err := h.apps.Mattermost.User.GetByUsername(username)
+	if err != nil {
+		return e(err, "get by username "+username)
+	}
+
+	fullMessage := md.Markdownf("Hallo! Your teammate %s sent you a message: \"%s\"", call.Context.ActingUserID, message)
+	h.DM(u.Id, fullMessage.String())
+	h.Ephemeral(call.Context.ActingUserID, call.Context.ChannelID, fmt.Sprintf("Sent \"%s\" to @%s", message, username))
+	httputils.WriteJSON(w,
+		apps.CallResponse{
+			Type:     apps.CallResponseTypeOK,
+			Markdown: md.Markdownf("Looks like you've submitted the form!"),
+		})
+
+	return 200, nil
+}
+
+func (h *helloapp) handleHelloCommandDefinition(w http.ResponseWriter, req *http.Request, claims *apps.JWTClaims, call *apps.Call) (int, error) {
+	data := SubCommand{
+		Pretext: "/hello",
+		AppID:   AppID,
+		SubCommands: []SubCommand{
+			{
+				AppID:       AppID,
+				Pretext:     "sendto",
+				FormURL:     PathCommandSubmission,
+				Description: "Send a message to someone in this channel",
+				Args: []interface{}{
```

Comment:
@levb This should probably be [`Autocomplete`](https://github.com/mattermost/mattermost-plugin-apps/blob/640b40537497565ab42778e2f0679afb7724e55e/server/apps/autocomplete.go#L3) or [`Form`](https://github.com/mattermost/mattermost-plugin-apps/blob/640b40537497565ab42778e2f0679afb7724e55e/server/apps/form.go#L3) instead of just an array of the form arguments.